Makefile

```diff
@@ -57,7 +57,8 @@ all: build/pyodide.asm.js \
   build/packages.json \
   build/test.html \
   build/webworker.js \
-  build/webworker_dev.js
+  build/webworker_dev.js \
+  download-fonts
```

Comment:
Maybe call this `copy`, instead of `download`?

Reply:
Yes, Sorry, fixed :)